### DISCUSSION TITLE
Wrong link when clicking on "matches in this file"

### DISCUSSION THREAD

**Discussion by WorldSEnder** [community/community#9743] [Other Feature Feedback, Questions, & Ideas] [answered]
Going to any file and selecting a recognized word, such as https://cs.github.com/yewstack/yew/blob/3ca7c0f90a551bdc0c9af51a2375114fc0c87d1f/examples/agents/src/lib.rs#L8 opens a box on the right side with helpful links to the Definition and "matches in this file". Clicking on any of those links in matches in this file redirects to the wrong page. Instead of going to the match, it jumps to the search front page.

I suppose this is simply a small oversight of how those links are constructed.

**Comment by colinwm** [comment, marked as answer]
Sorry about that! It's fixed now.
